.github/workflows/_build_test.yml

```diff
@@ -19,7 +22,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
```

Comment:
As described in the earlier review pass, running through Docker or NX parallel runs will only run in one instance of GitHub Actions, and we won't get the benefit of Github Action running in parallel. i.e. it will be a lot slower, so keeping this as it.

Reply:
Have you tested it? 

I could imagine it does not make a big difference, but I can be wrong. Standard Github runners [have 4 cores](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories). Most of our build tasks are not well parallelized and anecdotal I can tell that quite some time ™ is spent downloading things.

Reply:
Not that I want you to try it now or in the context of this PR, I just want to understand if this is something we might want to look into in the future.

Reply:
I do think it would be slower on a single machine even with 4 cores, and notable given that booting up the whole stack for each suite is not a trivial amount of resources. More fundamentally though I don't think we can parallel run test suites at this time due to port and proxy name conflicts of the services under test.

Reply:
Resolving it in the context of this PR.

Reply:
Thanks for your explanations.

Reply:
I had to try it ;)

You are right, it is slower.
4:36 without matrix https://github.com/wmde/wikibase-release-pipeline/actions/runs/10681174241
VS
3:#6 with matrix https://github.com/wmde/wikibase-release-pipeline/actions/runs/10680911492

Reply:
Ah, right. Sorry, I had lost track of the fact that we were talking about builds vs test. I’m fine with flattening this for now and eating the extra minute on each run. I don’t strongly prefer it but simplicity is nice, however I’d worry if we go towards turning-on multi-platform build with this setup that that little bit of extra might get compounded, so would rather leave it be for now.